server/cmwell-ws/src/test/logback-test.xml

```diff
@@ -0,0 +1,134 @@
+<configuration debug="false">
+
```

Comment:
add also the console appender, as you did with all the other logback-test files.